src/util/sss_semanage.c

```diff
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #ifdef HAVE_SEMANAGE
 #include <semanage/semanage.h>
+#include <selinux/selinux.h>
```

Comment:
After this change "selinux/selinux.h" is unused in child.

However "HAVE_SEMANAGE" != "HAVE_SELINUX". I have no idea why we have two conditions for it
because one without another does not make a sense.

Reply:
This is probably true. I personally see libselinux and libsemanage as one library, but it may be possible to have one installed and the other not, so it makes sense to have checks for both IMO and check them separately in the code as well.

But the unused header should indeed be removed.

Reply:
Please check my last comment about inlining